packages/amplify-category-api/src/provider-utils/awscloudformation/cdk-stack-builder/apigw-stack-transform.ts

```diff
@@ -217,8 +212,15 @@ export class ApigwStackTransform {
             external: true,
           },
         });
-
-        await sandboxNode.run(overrideCode, overrideJSFilePath).override(this.resourceTemplateObj as AmplifyApigwResourceStack);
+        try {
+          await sandboxNode.run(overrideCode, overrideJSFilePath).override(this.resourceTemplateObj as AmplifyApigwResourceStack);
+        } catch (err) {
+          throw new AmplifyError('InvalidOverrideError', {
+            message: `Executing overrides failed.`,
+            details: err.message,
+            resolution: 'There may be runtime errors in your overrides file. If so, fix the errors and try again.',
```

Comment:
We should probably print `overrideJSFilePath` in the error message. here and other places. Feel free to log this as follow up into https://app.asana.com/0/1202389680978480/1203201977550391/f .